### ISSUE TITLE
Remove annoyances

### ISSUE THREAD

**Issue by the-r3dacted** [Eclipse-Community/r3dfox-old#5]
Kinda a catch all issue as I might find more things, and I don't feel like creating separate issues for everything.

~~Disable pocket by default.
    Have decided I want to disable it by default but keep it available if wanted.~~ Done

~~Remove sponsored sites.
    Basically free advertising for whoever as no money is made for us.~~ Done

~~Enable userChrome and compact mode by default
    I assume enable those like I did the legacy taskbar backend in this commit. https://github.com/Eclipse-Community/r3dfox/commit/17e30a0732afd608f18ed12a6774adcb759a5917~~ Done

Tweaks from nara browser
    ~~Hide Firefox view. Just apply this commit to next release. https://github.com/aubymori/nara-browser/commit/7e37411ec8277f436309d8226f80974a3f3e583d~~ Decided against as this fully removes it.
    ~~These tweaks, private browsing not separate and view image info https://github.com/aubymori/nara-browser/commit/2e7b265420df329a2cc2072147b558fee195ca3c~~ Done
    ~~Add classic view image back to Firefox (not replace like the commit) https://github.com/aubymori/nara-browser/commit/75e0c68ac79ac62b6f2e1e2066370569ea402a1e~~ Seems hardcoded to be one or the other. Default behavior preferred so decided against.
    ~~Classic about:config page https://github.com/aubymori/nara-browser/commit/fecd46838073710d9bc858d56ece198be83cff78~~ Decided against as this may deviate a bit too far from stock and is a big commit.
    ~~Disable showSearchTerms https://github.com/aubymori/nara-browser/commit/d00b50c6c47332e3bd329adbd86e366798a9f156~~ Done
    More to come

~~Remove the setup step pages.
    Basically useless, does not need to exist.~~ Done

~~Remove default browser popup?
   Not sure if I want to do this since someone could forget, but it's also in the installer. If they untick it, they probably don't want it idk.~~ Decided against for now at least

**Comment by i486** [comment]
The advertisement for Firefox Focus appearing in the Private Window should be removed. It only appeared today.

![image](https://github.com/Eclipse-Community/r3dfox/assets/625325/0dd95272-9622-4742-b8d1-cc422f2c211e)

**Comment by the-r3dacted** [comment]
I haven't been able to replicate a Firefox Focus advertisement and I'm not sure where it would come from.

**Comment by the-r3dacted** [comment]
So I digged through the code, I found the advertisements in a selector, but no idea what calls or picks them. I just removed them all from the selector and it still seems to work.

**Comment by i486** [comment]
Just found this
```
Remove Focus promo in private tabs:
browser.promo.focus.enabled
```

**Comment by the-r3dacted** [comment]
Might revert current fix and just use that since it's a lot less lines.

**Comment by the-r3dacted** [comment]
Closing this as I can locally manage my stuffs. New issues can just be started.
